CHANGELOG.md

```diff
@@ -13,11 +13,37 @@ You can find its changes [documented below](#020-2024-09-19).
 
 ## [Unreleased]
 
-This release has an [MSRV] of 1.70.
+This release has an [MSRV] of 1.82.
 
 ### Changed
 
 - `Image` now stores the alpha as an `f32` ([#65][] by [@waywardmonkeys][])
+- Use `color` crate. See below for details. ([#63][] by [@waywardmonkeys][])
+
+### Color Changes
+
+The old code behind `peniko::Color` has been removed and color functionality is now provided by
+the [`color`] crate.
```

Comment:
In Markdown files let's keep to our [policy of one line per sentence](https://linebender.org/wiki/formatting-scheme/#markdown).

```suggestion
The old code behind `peniko::Color` has been removed and color functionality is now provided by the [`color`] crate.
```